src/app/grid/grid-groupby-summary-styling-sample/grid-groupby-summary-styling-sample.component.html

```diff
@@ -0,0 +1,19 @@
+<div class="summary-chooser">
+    <igx-buttongroup [values]="summaryCalcModes" (onSelect)="selectSummaryCalcMode($event)"></igx-buttongroup>
+</div>
+
+<div class="summary-chooser">
+    <igx-buttongroup [values]="summaryPositions" (onSelect)="selectSummaryPosition($event)"></igx-buttongroup>
+</div>
+
+<igx-grid #grid1 [data]="data" [width]="'100%'" [height]="'600px'" [rowSelectable]='true' [groupingExpressions]='expr' [showToolbar]="true" [columnPinning]="true">
+    <igx-column field="ShipCountry" header="Ship Country" width="200px" [groupable]="true" [pinned]="true">
+            <igx-icon class="pin-icon" fontSet="fas" name="fa-thumbtack" (click)="toggleColumn(col)"></igx-icon>
```

Comment:
THere is no toggleColumn method

Reply:
Remove the whole line or add a header template for the pin icon like in this topic - [https://staging.infragistics.com/products/ignite-ui-angular/angular/components/grid/column_pinning.html](https://staging.infragistics.com/products/ignite-ui-angular/angular/components/grid/column_pinning.html)

Reply:
> Remove the whole line or add a header template for the pin icon like in this topic - https://staging.infragistics.com/products/ignite-ui-angular/angular/components/grid/column_pinning.html

I've just removed that line, you can test it again : )